### PR TITLE
setting up

### PR DESCRIPTION
**Pull request by kylem038**
Refer to commits for general summary.

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -6,5 +6,14 @@
   </head>
   <body>
     <h1>Electron Color</h1>
+    <p>R<input class='red-value-input' type='number' min='0' max='255' step='1' /></p>
+    <p>G<input type='number' min='0' max='255' step='1' /></p>
+    <p>B<input type='number' min='0' max='255' step='1' /></p>
+    <p>RGB Value: rgba(<span class='red-value'></span>);</p>
+    <script src="jquery.js"></script>
+    <script>
+      try { $ = jQuery = module.exports; } catch(e) {}
```
Comment (blakeworsley) [inline app/index.html:16]: Good solution. In the future, I would like to see if we can move this to our installed dependency of jQuery.

File: app/index.html [hunk 2]
```diff
@@ -6,16 +6,11 @@
...
-    <p>RGBa Value: rgba(
-      <span class='red-value'></span>,
-      <span class='green-value'></span>,
-      <span class='blue-value'></span>,
-      <span class='alpha-value'></span>);
-    </p>
+    <p>R<input class='red-value-input' value='0' type='number' min='0' max='255' step='1' /></p>
+    <p>G<input class='green-value-input' value='0' type='number' min='0' max='255' step='1' /></p>
+    <p>B<input class='blue-value-input' value='0' type='number' min='0' max='255' step='1' /></p>
+    <p>A<input class='alpha-value-input' value='1' type='number' min='0' max='1' step='0.01' /></p>
```
Comment (blakeworsley) [inline app/index.html:12]: Glad we are stepping by 0.01 😄

File: package.json [hunk 3]
```diff
@@ -19,7 +19,8 @@
   },
   "homepage": "https://github.com/blakeworsley/electron-color#readme",
   "dependencies": {
-    "electron": "^1.4.10"
+    "electron": "^1.4.10",
+    "jquery": "^3.1.1"
```
Comment (blakeworsley) [inline package.json]: We should remove this dependency

**Review by blakeworsley — CHANGES_REQUESTED** [review]
Hey, just added a couple of questions. Otherwise, looks good.

File: package.json (hunk 3 above)
Comment (kylem038) [reply package.json]: Good fine - removed it.

**Review by blakeworsley — APPROVED** [review]

**Pull request merged**